go.mod

```diff
@@ -7,8 +7,8 @@ require (
 	contrib.go.opencensus.io/exporter/stackdriver v0.8.0
 	fortio.org/fortio v1.3.1
 	github.com/BurntSushi/toml v0.3.1 // indirect
+	github.com/ahmetb/gen-crd-api-reference-docs v0.1.1
```

Comment:
Just to clarify - these are now in here because we imported them in tools.go?

Reply:
Correct.